integration_test/README.md

```diff
@@ -29,8 +29,8 @@ the heron client:
 
 ```Bash
   bazel build --config=darwin integration_test/src/...
-  bazel run --config=darwin -- scripts/packages:heron-client-install.sh --user
-  bazel run --config=darwin -- scripts/packages:heron-tools-install.sh --user
+  bazel run --config=darwin -- scripts/packages:heron-install.sh --user
+  bazel run --config=darwin -- scripts/packages:heron-tests-tools-install.sh --user
```

Comment:
this change seems irrelevant to this pr?

Reply:
You're right, it's not relevant. I'll remove it.